src/nvim/drawline.c

```diff
@@ -400,40 +400,45 @@ static int get_sign_attrs(buf_T *buf, linenr_T lnum, SignTextAttrs *sattrs, int
 
 /// Prepare and build the 'statuscolumn' string for line "lnum" in window "wp".
 /// Fill "stcp" with the built status column string and attributes.
+/// This can be called three times per win_line(), once for virt_lines, once for
+/// the start of the buffer line "lnum" and once for the wrapped lines.
 ///
 /// @param[out] stcp  Status column attributes
 static void get_statuscol_str(win_T *wp, linenr_T lnum, int row, int startrow, int filler_lines,
                               int cul_attr, int sign_num_attr, int sign_cul_attr, char_u *extra,
```

Comment:
It seems that `extra` is unused in this function and `get_statuscol_display_info`

Reply:
Right, remnant from previous iteration of the original PR. Shouldn't this give a lint error?